### ISSUE TITLE
[BUG] Application created with wrong workload if there are more than one workloaddefinition

### ISSUE THREAD

**Issue by zzxwill** [cloud-native-application/RudrX#67]
There are more than one workloaddefinitions.
```
➜  /Users/zhouzhengxi/Programming/golang/src/github.com/cloud-native-application/RudrX git:(e2e) ✗ k get workloaddefinition
NAME                                  DEFINITION-NAME
containerizedworkloads.core.oam.dev   containerizedworkloads.core.oam.dev
deployments.apps                      deployments.apps
```

Run `rudr run containerized app-testworkloadrun-basic -p 80 nginx:1.9.4` to create an application. We found the expected workload wasn't containerized.
```
➜  /Users/zhouzhengxi/Programming/golang/src/github.com/cloud-native-application/RudrX git:(e2e) ✗ k get applicationconfiguration
NAME                        AGE
app-testworkloadrun-basic   39s
➜  /Users/zhouzhengxi/Programming/golang/src/github.com/cloud-native-application/RudrX git:(e2e) ✗ k get applicationconfiguration app-testworkloadrun-basic -p yaml
Error: unknown shorthand flag: 'p' in -p
See 'kubectl get --help' for usage.
➜  /Users/zhouzhengxi/Programming/golang/src/github.com/cloud-native-application/RudrX git:(e2e) ✗ k get applicationconfiguration app-testworkloadrun-basic -o yaml
apiVersion: core.oam.dev/v1alpha2
kind: ApplicationConfiguration
metadata:
  creationTimestamp: "2020-07-31T03:21:34Z"
  generation: 1
  name: app-testworkloadrun-basic
  namespace: default
  resourceVersion: "12355749"
  selfLink: /apis/core.oam.dev/v1alpha2/namespaces/default/applicationconfigurations/app-testworkloadrun-basic
  uid: 5ea82dd9-f5a7-413b-a7c2-1d09e6cdedc6
spec:
  components:
  - componentName: app-testworkloadrun-basic

➜  /Users/zhouzhengxi/Programming/golang/src/github.com/cloud-native-application/RudrX git:(e2e) ✗ k get component app-testworkloadrun-basic -o yaml
apiVersion: core.oam.dev/v1alpha2
kind: Component
metadata:
  creationTimestamp: "2020-07-31T03:21:34Z"
  generation: 1
  name: app-testworkloadrun-basic
  namespace: default
  resourceVersion: "12355747"
  selfLink: /apis/core.oam.dev/v1alpha2/namespaces/default/components/app-testworkloadrun-basic
  uid: 6de83a51-0b9f-4019-84b0-306d752dddd2
spec:
  workload:
    apiVersion: core.oam.dev/v1alpha2
    kind: deployment
    metadata:
      name: app-testworkloadrun-basic
    spec:
      containers:
      - image: nginx:1.9.4
        name: master
        ports:
        - containerPort: 80
          name: tbd
          protocol: TCP
status:
  latestRevision:
    name: app-testworkloadrun-basic-bshoshbd0cvka7locstg
    revision: 1
```

**Comment by erdun** [comment]
@wonderflow is refactoring run command mentioned in #63.

**Comment by wonderflow** [comment]
fixed by #70
